### ISSUE TITLE
8.0.0b transactions_get won't accept option include_original_description

### ISSUE THREAD

**Issue by imtibbet** [plaid/plaid-python#341]
We have been passing `include_original_description` to the transactions get endpoint on the old client ever since Plaid informed us that it was possible (it is not in the API documentation). However the new client really didn't want to accept the parameter, it kept raising a validation error when I tried to construct the request options. Here's the workaround I've come up with, but a less hacky option would be welcome:
```
options = TransactionsGetRequestOptions(
    count=batch,
    offset=offset,
    # include_original_description=True, # this doesn't work
)
request = TransactionsGetRequest(
    access_token,
    start_date,
    end_date,
    options=options,
)
request.options._data_store['include_original_description'] = True
client.transactions_get(request)
```

**Comment by phoenixy1** [comment]
@imtibbet `include_original_description` is a field that's currently available only in closed beta, which is why it's undocumented. Internally we're determining the tradeoff of exposing it in the OpenAPI file and/or docs, which would enable a less hacky way of using it in the beta client libraries, but might also cause confusion or frustration to developers who aren't in the beta. I've raised this to the team who owns this endpoint and they're going to come back with an answer on whether we can expose it officially yet.

**Comment by imtibbet** [comment]
FYI my hack works but the type of `original_description` in the `TransactionsGetResponse` object gets assigned dynamically, so a transaction with `original_description="2019-10-11"` will end up being a `datetime.date` property

**Comment by phoenixy1** [comment]
this should now be resolved with 8.0.0.b13. please reopen if not!
